projects/storefrontapp/tsconfig.app.json

```diff
@@ -4,8 +4,9 @@
     "module": "es2020",
     "outDir": "../../out-tsc/app",
     "strict": false, // TODO: Fix strict errors and remove this option
-    "types": [],
-    "target": "es2015"
+    "types": ["@sapui5/ts-types-esm"],
```

Comment:
Do we _really_ need `sap` types enabled globally? I hope we can avoid it, since we need it only in epd-visualisations lib (e.g. in visual-viewer.service). Could you look for other workarounds, please? Did you consider  declare the global `sap` object only locally in the file which uses it?

(same issue with tsconfig.server.json)

Reply:
`"types": ["@sapui5/ts-types-esm"],` is not needed. I will remove it.